src/objects/command.cc

```diff
@@ -87,53 +87,52 @@ std::ostream& operator<<(std::ostream& os, command const& obj) {
  *
  *  @return New command object.
  */
-command* add_command(char const* name, char const* value) {
-  // Make sure we have the data we need.
-  if (!name || !name[0] || !value || !value[0]) {
-    logger(log_config_error, basic)
-      << "Error: Command name or command line is NULL";
-    return (NULL);
-  }
-
-  // Check if the command already exist.
-  std::string id(name);
-  if (is_command_exist(id)) {
-    logger(log_config_error, basic)
-      << "Error: Command '" << name << "' has already been defined";
-    return (NULL);
-  }
-
-  // Allocate memory for the new command.
-  std::shared_ptr<command> obj(new command, deleter::command);
-  memset(obj.get(), 0, sizeof(*obj));
-
-  try {
-    // Duplicate vars.
-    obj->name = string::dup(name);
-    obj->command_line = string::dup(value);
-
-    // Add new items to the configuration state.
-    state::instance().commands()[id] = obj;
-
-    // Add new items to the list.
-    obj->next = command_list;
-    command_list = obj.get();
-
-    // Notify event broker.
-    timeval tv(get_broker_timestamp(NULL));
-    broker_command_data(
-      NEBTYPE_COMMAND_ADD,
-      NEBFLAG_NONE,
-      NEBATTR_NONE,
-      obj.get(),
-      &tv);
-  }
-  catch (...) {
-    obj.reset();
-  }
-
-  return (obj.get());
-}
+//command* add_command(char const* name, char const* value) {
+//  // Make sure we have the data we need.
+//  if (!name || !name[0] || !value || !value[0]) {
+//    logger(log_config_error, basic)
+//      << "Error: Command name or command line is NULL";
+//    return NULL;
+//  }
+//
+//  // Check if the command already exist.
+//  std::string id(name);
+//  if (is_command_exist(id)) {
+//    logger(log_config_error, basic)
+//      << "Error: Command '" << name << "' has already been defined";
+//    return NULL;
+//  }
+//
+//  // Allocate memory for the new command.
+//  std::shared_ptr<commands::command> obj(new commands::command);
+//
+//  try {
+//    // Duplicate vars.
+//    obj->set_name(name);
+//    obj->set_command_line(value);
+//
+//    // Add new items to the configuration state.
+//    state::instance().commands()[id] = obj;
+//
+//    // Add new items to the list.
+//    obj->next = command_list;
+//    command_list = obj.get();
+//
+//    // Notify event broker.
+//    timeval tv(get_broker_timestamp(NULL));
+//    broker_command_data(
+//      NEBTYPE_COMMAND_ADD,
+//      NEBFLAG_NONE,
+//      NEBATTR_NONE,
+//      obj.get(),
+//      &tv);
+//  }
+//  catch (...) {
+//    obj.reset();
+//  }
+//
+//  return obj.get();
+//}
```

Comment:
not needed anymore ? if yes you can delete this commented code.

Reply:
the file is dead